test/integration/api/integrations_controller_test.rb

```diff
@@ -170,4 +170,15 @@ def test_edit
     get edit_admin_service_integration_path(service_id: service.id)
     assert_response :not_found
   end
+
+  def
+    service = FactoryBot.create(:simple_service, account: @provider)
+    get admin_service_integration_path(service_id: service)
+    service.stubs(:oauth?).retuns(true)
+    assert_response :success
+    assert_match  "Example curl for testing", response.body
```

Comment:
This tests the label but not anything else inside the `unless`.

Extra tip: if it is just a string literal like this without interpolation or symbols like \n, you could use simple ' instead of ". For example: `'Example curl for testing'`